### PR TITLE
Adding Enfocado theme

### PR DESCRIPTION
**Pull request by codex0x09**
adding Enfocado theme (it contains four flavors),

### CODE REVIEW

**Comment by kchibisov** [comment]
What is the source of this theme? Just you created yourself?

**Comment by codex0x09** [comment]
- alacritty-enfocado theme's inspired form vim-enfocado (https://github.com/wuelnerdotexe).
- alacritty-enfocado (https://github.com/codex0x09/alacritty-enfocado) was wrote by codex0x09.
- there are some details should be in alacritty-enfocado repository.

**Comment by chrisduerr** [comment]
Could you link to the **original** source of the colors in the theme?

Also please don't link to your own repository, since that should be useless once this PR is merged, thus making it an unnecessary point of failure.

**Comment by codex0x09** [comment]
it was inspired from vim-enfocado https://github.com/wuelnerdotexe/vim-enfocado

**Comment by chrisduerr** [comment]
> it was inspired from vim-enfocado https://github.com/wuelnerdotexe/vim-enfocado

Then document that as the source.

**Comment by codex0x09** [comment]
I apologize for being late, my connection is not good.
I'v changed the source in the each theme file to vim-enfocado repo,
but if any one see the source and than go there, they wont find the source code of the alacritty-enfocado theme.
could you teach me why this ?

**Comment by chrisduerr** [comment]
> they wont find the source code of the alacritty-enfocado theme

They're looking at the markup of the theme when reading it in this repo. It's the sole truth as far as Alacritty themes go.
The only thing they might be curious about is what these colors are supposed to be based on.

File: themes/enfocado_neon_dark.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# Normal colors
+[colors.normal]
+black   = '#4a4a4a'
+red     = '#ed4a46'
+green   = '#70b433'
+yellow  = '#ee5400'
+blue    = '#a572d2'
+magenta = '#eb4eb7'
+cyan    = '#56b8c9'
```
Comment (chrisduerr) [inline themes/enfocado_neon_dark.toml:22]: How did you arrive at these colors? They do not match the enfocado theme source.

**Review by chrisduerr — CHANGES_REQUESTED** [review]

**Comment by codex0x09** [comment]
> > they wont find the source code of the alacritty-enfocado theme
> 
> They're looking at the markup of the theme when reading it in this repo. It's the sole truth as far as Alacritty themes go. The only thing they might be curious about is what these colors are supposed to be based on.

I can't thank you enough for all your patience and giving to me valuable information.
Thank you so much.

File: themes/enfocado_neon_dark.toml (hunk 1 above)
Comment (codex0x09) [reply themes/enfocado_neon_dark.toml:22]: I've modified those color, I thought in this way they become good!
if not or  they should not be modified, than I'm going to back them as they was.

Comment (codex0x09) [reply themes/enfocado_neon_dark.toml:22]: I've no idea, what should I do ?

Comment (chrisduerr) [reply themes/enfocado_neon_dark.toml:22]: The goal of these shared themes is to give people easy access to popular themes. If you modify themes to be your personal ideal colorscheme instead then the potential userbase for that is exactly 1 person, at which point there's no reason for us to host that theme.

So generally speaking the original colors should be used, so when someone interested in the enfocado theme looks through this repository, it's actually what they get.

Comment (codex0x09) [reply themes/enfocado_neon_dark.toml:22]: I'm going to back theme(enfocado_neon*), but enfocado's author replace the blue colors with the violet colors !!
I note that: he set the blue one of the main colors in the (enfocado_nature*),
while the violet takes the place of blue in (enfocado_neon*).
based on this I'm going re-set the color to match the original theme.

**Comment by codex0x09** [comment]
I'm sorry, there are many commits. I'm trying to keep it simple but...
I hope now it's become as it should be !

File: README.md [hunk 2]
```diff
@@ -68,6 +68,10 @@ configuration file.
...
 |                                                **_dracula_plus_**<br>[source](https://draculatheme.com)                                           |               ![dracula_plus](images/dracula_plus.png)     |
+|                                                **_enfocado_nature_dark_**<br>[source](https://github.com/wuelnerdotexe/vim-enfocado.git)          |![enfocado_nature_dark](images/enfocado_nature_dark.png)    |
+|                                                **_enfocado_nature_light_**<br>[source](https://github.com/wuelnerdotexe/vim-enfocado.git)         |![enfocado_nature_light](images/enfocado_nature_light.png)  |
+|                                                **_enfocado_neon_dark_**<br>[source](https://github.com/wuelnerdotexe/vim-enfocado.git)            |![enfocado_neon_dark](images/enfocado_neon_dark.png)        |
+|                                                **_enfocado_neon_light_**<br>[source](https://github.com/wuelnerdotexe/vim-enfocado.git)           |![enfocado_neon_light](images/enfocado_neon_light.png)      |
```
Comment (chrisduerr) [inline README.md:74]: Where did you get the names of these colorschemes from? From the theme's readme it seems like both nature and neon have the same colors, but they just use them for different syntax tokens (which is irrelevant for Alacritty).

File: themes/enfocado_nature_dark.toml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+[colors.primary]
+background =  '#181818'
+foreground =  '#b9b9b9'
+
+# Normal colors
+[colors.normal]
+black   = '#3b3b3b'
+red     = '#ed4a46'
+green   = '#70b433'
+yellow  = '#dbb32b'
```
Comment (chrisduerr) [inline themes/enfocado_nature_dark.toml:14]:
```suggestion
yellow  = '#dbb32d'
```

Typo?

File: themes/enfocado_nature_dark.toml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+blue    = '#368aeb'
+magenta = '#eb4eb7'
```
Comment (chrisduerr) [inline themes/enfocado_nature_dark.toml:16]: Shouldn't this be `#eb6eb7`?

File: themes/enfocado_nature_dark.toml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+cyan    = '#56b8c9'
```
Comment (chrisduerr) [inline themes/enfocado_nature_dark.toml:17]: Shouldn't this be `#3fc5b7`?

File: themes/enfocado_nature_dark.toml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+white   = '#b9b9b9'
+
+# Bright colors
+[colors.bright]
+black   = '#777777'
+red     = '#ff5e56'
+green   = '#83c746'
+yellow  = '#efc541'
+blue    = '#4f9cf3'
```
Comment (chrisduerr) [inline themes/enfocado_nature_dark.toml:26]:
```suggestion
blue    = '#4f9cfe'
```

File: themes/enfocado_nature_dark.toml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+magenta = '#fb4fb7'
```
Comment (chrisduerr) [inline themes/enfocado_nature_dark.toml:27]:
```suggestion
magenta = '#ff81ca'
```

File: themes/enfocado_nature_dark.toml [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+cyan    = '#3fc5b7'
```
Comment (chrisduerr) [inline themes/enfocado_nature_dark.toml:28]:
```suggestion
cyan    = '#56d8c9'
```

**Review by chrisduerr — CHANGES_REQUESTED** [review]
I've only gone through and checked the colors of the dark theme, of which several were not matching upstream.

I'm not sure what the problem is here, it shouldn't be that difficult to copy some hex colors.

**Comment by codex0x09** [comment]
> I've only gone through and checked the colors of the dark theme, of which several were not matching upstream.
> 
> I'm not sure what the problem is here, it shouldn't be that difficult to copy some hex colors.

I've made many changes in the original colors when I did PR,
when you've checked the dark theme, I've removed all my change even on that you not talk about yet, which are the other dark and the light themes.
I fill that I've broken things or made many mistakes, 
if it possible, remove my PR and I'm going to do all your recommendations and then
I'll do PR  again !,

File: README.md (hunk 2 above)
Comment (codex0x09) [reply README.md:74]: The name 'alacritty-enfocado' is inspired form 'vim-enfocado' which is this theme for Alacritty terminal not made by alacritty, this was my thought !.

I thought that (nature-dark and neon-dark) are different as well as the lights,
based on the images in the enfocado's readme, this everything.

No problem, it's possible to change names or even removing neon-[dark | light], as long as you see that right and should be.

File: themes/enfocado_nature_dark.toml (hunk 3 above)
Comment (codex0x09) [reply themes/enfocado_nature_dark.toml:14]: I apologize for that, I've removed everything(enfocado theme) from master branch and then rewrite the colors again in another branch, it was mistake.

File: themes/enfocado_nature_dark.toml (hunk 4 above)
Comment (codex0x09) [reply themes/enfocado_nature_dark.toml:16]: yes you're right, I've made mistake here

File: README.md (hunk 2 above)
Comment (chrisduerr) [reply README.md:74]: Considering that as far as I can tell the enfocado theme just has different colors for different syntax tokens with the two different themes, I think it makes sense to only have on enfocado dark/light theme, not two of each.

Comment (codex0x09) [reply README.md:74]: Okay no problem,
which tow should I remove (one of dark and light themes )?!!
what is about the name alacritty-enfocado ?
the theme name will become enfocado-dark and enfocado-light !! as you've mentioned

Comment (codex0x09) [reply README.md:74]: I agree with you.

Comment (codex0x09) [reply README.md:74]: I apologize for being late.

**Comment by codex0x09** [comment]
Now, Enfocado theme has only tow mode dark / light.

File: themes/enfocado_dark.toml [hunk 9]
```diff
@@ -0,0 +1,29 @@
+# Theme: enfocado_nature_dark
```
Comment (chrisduerr) [inline themes/enfocado_dark.toml:1]: There's still a reference to the old theme name here.

File: themes/enfocado_light.toml [hunk 10]
```diff
@@ -0,0 +1,29 @@
+# Theme: enfocado_nature_light
```
Comment (chrisduerr) [inline themes/enfocado_light.toml:1]: There's still a reference to the old theme name here.

**Review by chrisduerr — CHANGES_REQUESTED** [review]
Just some minor changes left, otherwise everything looks good.

File: themes/enfocado_light.toml (hunk 10 above)
Comment (codex0x09) [reply themes/enfocado_light.toml]: Oops, I'm sorry about that

**Comment by codex0x09** [comment]
Mr @chrisduerr 
Thank you from bottom of my heart, I learn a lot from you (>^.^<).

**Review by chrisduerr — APPROVED** [review]

**Pull request merged**